### PR TITLE
chore: reorganise inspector / sidebar

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Prompted by a bunch of different things, including:
- Work to add a new "stepped debugging" experience in replay
- Suggestion that the current view mode tabs are confusing in https://github.com/PostHog/posthog/issues/24561
- Idea to add a new "Session / User" panel to recordings

We need to restructure the current view to give more space

## Changes

Bringing the right hand sidebar back...

sidebar

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Tried out a lot of different combinations but please do pull it down and test things out for yourself

### CODE REVIEW

File: frontend/src/scenes/persons/PersonDisplay.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ export interface PersonDisplayProps {
     noEllipsis?: boolean
     noPopover?: boolean
     isCentered?: boolean
+    children?: React.ReactChild
```
Comment (pauldambra) [inline frontend/src/scenes/persons/PersonDisplay.tsx:26]: Silly question... ELI5

Why not import { ReactChild } above and then use it without the `React.` here?

Comment (daibhin) [reply frontend/src/scenes/persons/PersonDisplay.tsx:26]: Tbh I have no valid reason to use one over the other 🤷 I guess the `React` global is available so I just start writing and it doesn't auto import

Quick scan of the codebase suggests that's how it's been done elsewhere
Screenshot 2024-09-12 at 20 54 49

**Comment by pauldambra** [comment]
locally i had neither playback nor waterfall selected and it defaulted to waterfall but had no nav data. need to make sure it defaults to playback

Screenshot 2024-09-12 at 22 10 36
Screenshot 2024-09-12 at 22 10 41

i'm not super-jazzed about the placement of waterfall here but it anyway needs way more work and has terrible retention so we'll survive

**Comment by pauldambra** [comment]
i do like being able to have the sidebar and the waterfall view at the same time though

Screenshot 2024-09-12 at 22 12 41

**Comment by pauldambra** [comment]
ah! i don't even have network capture enabled locally 🤔 something for the future but that's not discoverable and somewhere we could upsell

Screenshot 2024-09-12 at 22 14 21

**Comment by pauldambra** [comment]
Screenshot 2024-09-12 at 22 17 20

those two icons are pretty small, could they have less padding - and a tooltip

**Comment by pauldambra** [comment]
width of the panel is jumping around as i change tab

**Review by pauldambra — COMMENTED** [review]
bunch of nits but this is great

i think waterfall needs better positioning but it also needs work so we don't need to solve that now

**Comment by daibhin** [comment]
@pauldambra all done except for the last issue with the sidebar width changing - couldn't manage to reproduce that locally 🤔 Agreed we need to find somewhere for waterfall to live... I have some thoughts but would rather hold off until we do an overall review to save on implementation churn

**Review by pauldambra — APPROVED** [review]
> Agreed we need to find somewhere for waterfall to live...

yep, it needs focus and we've too much else on right now - and anyway if folk complain about it moving we know it needs focus :)

happy to do a fast follow for sidebar width if it's data dependent and not my machine dependent :)
